cmd/dataset_tokenizer/dataset_tokenizer.go

```diff
@@ -166,7 +166,7 @@ func ReadTexts(dirPath string, sanitize bool, sortSpec string) (TextsIterator,
 	}
 
 	if sortSpec != "" {
-		if sortSpec == "size_ascending" {
+		if sortSpec == "size_ascending" || sortSpec == "shuffle" {
```

Comment:
Why are we sorting files by size on a shuffle? If we're shuffling, the ordering of files doesn't matter?

Reply:
it was to make sure we had a fixed order of file paths to compare it to, but i'm pretty sure none will still be fixed so ill change that